src/components/Dashboard/TodoList.vue

```diff
@@ -1075,6 +1075,7 @@ export default class TodoList extends Mixins(
         correctedFilingId: correction.correctedFilingId,
         // this is only used for external corrections (IA):
         correctedFilingType: this.filingTypeToName(correction.correctedFilingType as FilingTypes),
+        type: correction.type,
```

Comment:
What is this change for?

If this change is needed, please add this property to the Todo Item interface.